src/appendix/bibliography.md

```diff
@@ -89,3 +89,4 @@ Rust, as well as publications about Rust.
 * [The Case for Writing a Kernel in Rust](https://www.tockos.org/assets/papers/rust-kernel-apsys2017.pdf)
 * [RustBelt: Securing the Foundations of the Rust Programming Language](https://plv.mpi-sws.org/rustbelt/popl18/)
 * [Oxide: The Essence of Rust](https://arxiv.org/abs/1903.00982). By Aaron Weiss, Olek Gierczak, Daniel Patterson, Nicholas D. Matsakis, and Amal Ahmed.
+* [Polymorphisation: Improving Rust compilation timesthrough intelligent monomorphisation](https://davidtw.co/media/masters_dissertation.pdf)/ By David Wood
```

Comment:
nit:

```suggestion
* [Polymorphisation: Improving Rust compilation timesthrough intelligent monomorphisation](https://davidtw.co/media/masters_dissertation.pdf). David Wood's master's thesis.
```

Reply:
Also, there's a missing space between `times` and `through`.